core/src/test/scala/unit/kafka/utils/TestUtils.scala

```diff
@@ -1135,6 +1136,71 @@ object TestUtils extends Logging {
   }
 
   /**
+   * Verifies that this ACL is the secure one. The
+   * values are based on the constants used in the
+   * ZooKeeper code base.
```

Comment:
It would be nice if we linked to these constants in the ZK code base or at least explained them. Another existing issue, but it's quite difficult to understand what these magic values mean. :)